lab-matt/app/html/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html ng-app='adventureApp'>
+  <head>
+    <meta charset="utf-8">
+    <title>SPACE... ADVENTURE... SCIENCE</title>
+    <link rel="stylesheet" type= "text/css" href="bundle.css">
+    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+    <script src = 'bundle.js'></script>
+  </head>
+  <body>
+      <main ng-controller="GameController as GC">
+        <form novalidate name="gameForm">
+          <select ng-model="GC.gameForm.direction">
```

Comment:
If I were doing this, I would change this to:
`<select ng-init="gameForm.direction='North'" ng-model="gameForm.direction">`
and change all `GC.gameForm.direction` to just `gameForm.direction`